packages/amplify_datastore/ios/Classes/types/model/FlutterModelField.swift

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import Flutter
+import Foundation
+import Amplify
+
+public struct FlutterModelField {
+    
+    public let name: String
+    public let type: ModelFieldType
+    public let isRequired: Bool
+    public let isArray: Bool
+    public let attributes: [ModelFieldAttribute] = []
+    public let association: ModelAssociation? = nil
+    public let authRules: AuthRules = []
+    
+    init(serializedData: [String: Any]) {
+        self.name = serializedData["name"] as! String
+        self.type = ModelFieldType.from(type: FlutterModelField.deserializeFieldType(type: serializedData["type"] as! String))
+        self.isRequired = serializedData["isRequired"] as! Bool
```

Comment:
This (or the equivalent configure()) method is not part of this PR I just put together something to enable me to test since these fields are schema is not final. CC @fjnoyp to consider this when he implements configure for iOS